sources/PTYTextView.m

```diff
@@ -1434,7 +1435,8 @@ - (void)keyDown:(NSEvent*)event {
         // track the instance of PTYTextView that is currently handling a key event and rerouting
         // calls as needed in -insertText and -doCommandBySelector.
         gCurrentKeyEventTextView = [[self retain] autorelease];
-        [self interpretKeyEvents:[NSArray arrayWithObject:event]];
+        _keydownEvent = event;
+        _inputMethodIsInserting = !event.isARepeat && [self.inputContext handleEvent:event];
```

Comment:
Are you about to reproduce the problem with key repeat? On OS 10.11 it doesn't seem to happen.